src/config/_screens.scss

```diff
@@ -3,6 +3,14 @@
 @import 'config/variables';
 
 // Declarations
+
+// Variables
+$screen-desktop: #{map-get(map-get($variables, 'screens'), 'screen-desktop')}px;
+$screen-tablet: #{map-get(map-get($variables, 'screens'), 'screen-tablet')}px;
+$screen-mobile: #{map-get(map-get($variables, 'screens'), 'screen-mobile')}px;
+$screen-sm-mobile: #{map-get(map-get($variables, 'screens'), 'screen-sm-mobile')}px;
```

Comment:
@devowhippit  should this also be removed so we only use the mixin?

Reply:
@KISS This looks fine. Which mixin are you referring to?

Reply:
This `@mixin screens($screens: map-get($variables, 'screens')) {..}`. I'm confused about when and how we use mixins > variables and the use of `${map-get}`. Don't need to run through now, added to the list of things to cover in the patterns meeting :)

Reply:
Oh gotcha. The screens mixin provides a nice way to create utilities that will apply to all screen sizes. An example is in the [icons stylesheet](https://github.com/CityOfNewYork/nyco-patterns/blob/c8e7df4ed8030d9c3867c8f1b0505b25aa1e51a2/src/elements/icons/_icons.scss#L55). Say I want to have an icon and I want the mobile first size to be something like `icon-1`. However, for desktop I want it to be larger so I can use the utility `icon-1 screen-desktop:icon-4`. The screens mixin creates the responsive utilities (`screen-desktop:`) based on all of the configured screens.

Using `map-get($variables, 'something);` as opposed to creating a shorthand `$something` for the variable is your preference so I think this is fine. I generally like to reference `map-get($variables, 'something);` because I don't want to create shorthands every time I create a  but nested variable in `$variables` but working with shorthands can be easier... as long as we make sure the match the variable they are referencing!!!

Reply:
The screens mixin may not be necessary at all however, because tailwind provides this functionality by allowing you to extend their library with your own utilities; https://tailwindcss.com/docs/adding-new-utilities